src/client/content_mapblock.cpp

```diff
@@ -1039,8 +1075,23 @@ void MapblockMeshGenerator::drawPlantlikeNode()
 
 void MapblockMeshGenerator::drawPlantlikeRootedNode()
 {
+	u8 facedir = 0;
+
+	if (f->param_type_2 == CPT2_FACEDIR ||
+			f->param_type_2 == CPT2_COLORED_FACEDIR) {
+		facedir = n.getFaceDir(nodedef);
+	} else if (f->param_type_2 == CPT2_WALLMOUNTED ||
+			f->param_type_2 == CPT2_COLORED_WALLMOUNTED) {
```

Comment:
This part of code allows something like this to be shown:
![screenshot_20210620_060041](https://user-images.githubusercontent.com/17455197/122662000-f7a13d80-d18f-11eb-953c-de5783792d8b.png)

May be useful for some decorations. Because of it, I kept this part of copied code.
But if you see it useless, I will remove it.

Reply:
My bad, this is a different part of code.

So fixed.